server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -851,11 +851,9 @@ def _delete_model_endpoint_monitoring_infra(
             ModelEndpoints.delete_tsdb_records,
             mlrun.mlconf.background_tasks.default_timeouts.operations.model_endpoint_tsdb_leftovers,
             background_task_name,
+            None,
             project,
             uids,
```

Comment:
just a recommendation - I think it's more clear in that approach
```suggestion
            project=project,
            uids=uids,
```